sys/log/log_printfnoformat/log_module.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_log_printfnoformat puts log module
+ * @ingroup     sys
+ * @brief       This module implements an example logging module
+ * @{
+ *
+ * @file        log_module.h
+ * @brief       log_module header
+ *
+ * @author      Jason Linehan <patientulysses@gmail.com>
+ * @author      Christian Mehlis <mehlis@inf.fu-berlin.de>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef __LOG_FORMAT_H
+#define __LOG_FORMAT_H
+
+#include <stdio.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief log_write overridden function
+ *
+ * This example function will only print the log's format string.
+ * Use it where printf might be too heavy.
+ *
+ * @param[in] level (unused)
+ * @param[in] format String that the function will print
+ */
+static inline void log_write(unsigned level, const char *format, ...) {
+    (void)level;
+    puts(format);
+}
```

Comment:
Building application "foobar" for "msb-430h" with MCU "msp430f1612".
> What about MSP430?

Apparently not:
```
/home/oleg/git/miniature-dangerzone/playground/playground.c:1:20: fatal error: unistd.h: No such file or directory
...
/home/oleg/git/miniature-dangerzone/playground/bin/msb-430h/foobar.a(playground.o): In function `main'
/home/oleg/git/miniature-dangerzone/playground/playground.c:4: undefined reference to `write'
collect2: ld returned 1 exit status
```